htdocs/mri/jiv/get_file.php

```diff
@@ -123,6 +123,26 @@
     $MimeType         = 'application/xml';
     $DownloadFilename = basename($File);
     break;
+case 'edf':
```

Comment:
@johnsaigle let me know if those modifications of the get_file.php file is not correct.

Reply:
I think this looks fine. 👍

Reply:
based on my experience with EDF, there may be an additional modification to the bottom of this file to facilitate large downloads. 
- [ ] Note to self @christinerogers to come back to this